docs/common/yum.md

```diff
@@ -1,46 +1,87 @@
-YUM Repositories
+OSG Yum Repositories
 ====================
 
-About This Document
--------------------
-
-This document introduces YUM repositories and how OSG uses them.
-If you are unfamiliar with YUM, see the [documentation on using YUM and RPM](/release/yum-basics).
+This document introduces Yum repositories and how they are used in the OSG.
+If you are unfamiliar with Yum, see the [documentation on using Yum and RPM](/release/yum-basics).
 
 Repositories
 ------------
 
-OSG hosts four public-facing repositories at [repo.opensciencegrid.org](https://repo.opensciencegrid.org/):
+The OSG hosts multiple repositories at [repo.opensciencegrid.org](https://repo.opensciencegrid.org/osg/) that are
+intended for public use:
 
--   **release**: RPMs considered production-ready.
--   **testing**: RPMs not yet ready for release; expect bugs.
--   **development**: RPMs that are bleeding-edge;
-      do not use without instruction from OSG Software and Release team members.
--   **contrib**: RPMs contributed from outside the OSG S&R team;
-      no official OSG support.
--   **upcoming release/testing/development**: Similar to release/testing/development but for new versions of software that may require manual action after an update.
+| Yum repository                                                                             | Contains RPMs that...                                                                                                                |
+|--------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------|
+| `osg`                                                                                      | Are considered production-ready (default).                                                                                           |
+| `osg-rolling`                                                                              | Passed integration and acceptance testing but have not yet been officially released.                                                 |
```

Comment:
We won't untag anything from the rolling repos when we put them in a numbered release; it will be an alternate release stream. All the RPMs in there are considered production-ready. How about something like "are considered production-ready but released at a faster pace than the ones in the `osg` repos"?

Reply:
I like that a lot better. Updated.